src/main/webapp/app/course/manage/course-group.component.ts

```diff
@@ -16,6 +16,11 @@ import { DataTableComponent } from 'app/shared/data-table/data-table.component';
 import { iconsAsHTML } from 'app/utils/icons.utils';
 import { AccountService } from 'app/core/auth/account.service';
 import { EventManager } from 'app/core/util/event-manager.service';
+import { ExportToCsv } from 'export-to-csv';
+
+export const NAME_KEY = 'Name';
+export const USERNAME_KEY = 'Username';
+export const EMAIL_KEY = 'Email';
```

Comment:
Is the `export` needed?

Reply:
I didn't want to differ from other pages that use these variables as export.

Reply:
Solved